s2apler/text.py

```diff
@@ -472,7 +475,10 @@ def normalize_text(text: Optional[str], special_case_apostrophes_and_dashes: boo
     # if there is the possibility of latex
     # we can convert it to text
     if "\\" in text or text.count("$") > 1:
-        text = latex_to_text(text)
+        try:
```

Comment:
Are there common problems we can try to fix here too? Like find all `\href` with regex and remove prior to converting?